app/src/main/java/org/simple/clinic/sync/SyncConfigType.kt

```diff
@@ -0,0 +1,11 @@
+package org.simple.clinic.sync
+
+import javax.inject.Qualifier
+
+@Qualifier
+annotation class SyncConfigType(val type: Type) {
+
+  enum class Type {
+    Drugs
```

Comment:
I'm guessing we would also move `Daily` and `Frequent` here, correct?

Reply:
Yep, will be done in a different PR eventually